Dockerfile

```diff
@@ -1,43 +1,64 @@
-FROM alpine:3.7
+FROM alpine:3.8 AS base
```

Comment:
AFAICT this will run as `root`, ideally we'd want this to run as a normal user.

Reply:
👍